app/templates/components/currency-amount.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if (gt @amount 0)}}
+    {{currency-symbol @currency}}
+{{/if}}
+
+{{#if (eq @format false)}}
```

Comment:
There is no case amount shouldn't be formatted

Reply:
there is a place in ticket overview page where amount was not being formated so acc to that but if that was a bug? that I will fix this